### ISSUE TITLE
Fenced code blocks not recognized when nested under list item

### ISSUE THREAD

**Issue by dbader** [DavidAnson/markdownlint#152]
Thanks for this awesome project, it's been incredibly helpful in standardizing the format of our written content 😄 

I'm having an issue with fenced code blocks that are nested under list items not being recognized as code blocks. For example, the following file will trigger the `no-multiple-blanks` while inside the second code block.

`test.md`:

    # Test

    ```text
    line1


    line2
    ```

    - List item 1

        ```text
        line1


        line2
        ```

```sh
$ markdownlint test.md
test.md: 15: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
```

Running `markdownlint@0.11.0` via `markdownlint-cli@0.13.0`

**Comment by DavidAnson** [comment]
Sorry about that, I think this is a duplicate of what’s underlying #151. I will look into this once I finish a couple of other in-progress projects. Thanks!

**Comment by dbader** [comment]
No worries at all David and THANKS SO MUCH for this awesome project, it's been incredibly helpful :)
